maven/lib/dependabot/maven/update_checker/version_finder.rb

```diff
@@ -58,7 +58,7 @@ def versions
             repositories.map do |repository_details|
               url = repository_details.fetch("url")
               xml = dependency_metadata(repository_details)
-              next [] if xml.blank?
+              next [] if xml.nil? || xml.blank?
```

Comment:
The `nil?` part can be undone in the end I guess?

Reply:
Uh yeah, what happened here? 😆

Reply:
Oh ok, so ActiveSupport seems to have been providing a nil-safe `blank?`, but there's also a Nokogiri `blank?` that isn't nil-safe. So I think we can just just for `nil?` here.

Reply:
Mmmm, I guess if we need to deal with [blank? XML nodes](https://www.rubydoc.info/github/sparklemotion/nokogiri/Nokogiri%2FXML%2FNode:blank%3F) here, then the current version adding back nil safety seems fine?

Reply:
I don't think it matters since it only calls `css` with a selector that it won't find, should have the same result either way.

```ruby
irb(main):006:0> Nokogiri.XML("not even valid").css("no > exist")
=> []
```